src/attempt/validators.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from "zod";
+import { validateHandlerRequest } from "../services/server/validators";
+
+export const getAttemptsByQuizIdRequestSchema = z.object({
+    params: z.object({
+        quizId: z.string(),
+    }),
```

Comment:
shouldnt get in query? in rest the id in path supposed to the attemptsId. for this route by rest api. there suppose to be route for getting list of attemps (GET /attempts) whith query of filters.
for example https://......./attempts?quizId=758438-12478234-1324
```suggestion
    query: z.object({
        quizId: z.string(),
    }),
```